packages/component-library/package.json

```diff
@@ -74,36 +75,26 @@
   },
   "dependencies": {
     "@material-ui/core": "^4.1.3",
-    "chai-enzyme": "^1.0.0-beta.1",
```

Comment:
This package is being used in `mocha.conf.js` which is used for testing
https://github.com/hackoregon/civic/blob/master/packages/component-library/mocha.conf.js#L50

Reply:
Ah ha! This is added to `devDependencies`. Nothing to see here.